.github/workflows/cli-rc.yml

```diff
@@ -59,6 +59,13 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm install
+        working-directory: ./src/dependency-manager
+      - run: npm run build
```

Comment:
why do we build - remove and build again?

Reply:
If this is some kind of workaround, seems like we'd be better off updating the npm build command to do what we want instead of working around it in the CI job.

Reply:
The `npm run build`s are two different builds, one is the dep-manager and one is the cli.

We remove `node_modules` from the dep-manager after we're done with it because the CLI tests do not pass when there are any `node_modules` in the dep-manager. This is an issue that dates back beyond last fall. If memory serves correctly I think @davidthor also pointed it out one of our daily standups semi-recently, maybe a month back.

You can reproduce the issue locally: `cd src/dependency-manager`, `npm install`, `cd ../..`, `npm run test` (should fail). then `rm -rf src/dependency-manager/node_modules`, `npm run test` (should pass).

I'm not sure it's really a workaround that belongs in the `npm run build` script.  I've opened an issue at the very least so we can track it: https://gitlab.com/architect-io/architect-cli/-/issues/93

Reply:
I get it now - just confusing to look at